artifactory/utils/container/buildinfo.go

```diff
@@ -91,10 +92,23 @@ func (builder *buildInfoBuilder) getSearchableRepo() string {
 
 // Set build properties on image layers in Artifactory.
 func setBuildProperties(buildName, buildNumber, project string, imageLayers []utils.ResultItem, serviceManager artifactory.ArtifactoryServicesManager) (err error) {
+	// Skip if no build info is provided
+	if buildName == "" || buildNumber == "" {
+		log.Debug("Skipping setting properties - no build name or build number provided")
```

Comment:
```suggestion
		log.Debug("Skipping setting properties - build name and build number are required")
```
updated debug log message